ktor-client/ktor-client-plugins/ktor-client-logging/jvm/src/io/ktor/client/plugins/logging/LoggerJvm.kt

```diff
@@ -16,11 +17,37 @@ public actual val Logger.Companion.DEFAULT: Logger
     }
 
 /**
- * Android [Logger]: breaks up long log messages that would be truncated by Android's max log
- * length of 4068 characters
+ * Android [Logger]: Logs to the Logcat on Android if the SLF4J provider isn't found.
+ * Otherwise, uses the [Logger.Companion.DEFAULT].
+ * Breaks up long log messages that would be truncated by Android's max log
+ * length of 4068 characters.
  */
 public val Logger.Companion.ANDROID: Logger
-    get() = MessageLengthLimitingLogger()
+    get() {
+        val logger = Logger.DEFAULT
```

Comment:
please consider making an internal object to avoid multiple initializations

Reply:
What do you mean? In the body of `get` the default logger is created only once.

Reply:
It will be new every get call